.github/CODEOWNERS

```diff
@@ -907,6 +907,9 @@
 # ServiceLabel: %WebPubSub
 # ServiceOwners:                                                   @vicancy @JialinXin
 
+# ServiceLabel: %HardwareSecurityModules
```

Comment:
@ronniegeraghty:  Looks like there's a proposed new tag.   I'm going to suggest that we add spaces for readability:  "Hardware Security Modules".

Please offer thoughts.  If approved, I'll push it out to all repositories.

Reply:
updated service Label to "Hardware Security Modules"